upa/src/tool/devAggregator.ts

```diff
@@ -215,12 +214,12 @@ async function submitBatch(
 
   // Submit aggregated proof
   await upaInstance.verifier.verifyMixedAggregatedProof(
+    Uint8Array.from(aggProofParams.dupSubmissionIdxs),
     calldata,
     proofIds,
     aggProofParams.numOffChainProofs,
     aggProofParams.submissionProofs,
     packOffChainSubmissionMarkers(aggProofParams.offChainSubmissionMarkers),
-    packDupSubmissionIdxs(aggProofParams.dupSubmissionIdxs),
```

Comment:
Can we remove this function from the SDK?  (I may have missed the diff removing it)